api_types.ts

```diff
@@ -1084,3 +1084,19 @@ export interface SyncCompletionMetadata {
    */
   hasIncompleteResults?: boolean;
 }
+
+/**
+ * TODO(patrick): Unhide this
+ * @hidden
+ */
+export interface ItemMatchingPredicate {
+  /**
+   * The name of the property to search for matches.
+   */
+  property: string;
+
+  /**
+   * What value(s) to search for.
+   */
+  matches: string | string[] | number | number[] | boolean | boolean[];
```

Comment:
By default do we assume these are exact matches? Assuming yes and that if we want regex support we can add something else

Reply:
Exact matches for now, yeah